superset-frontend/src/components/Alert/Alert.stories.tsx

```diff
@@ -82,6 +84,7 @@ InteractiveAlert.args = {
   message: smallText,
   description: bigText,
   showIcon: true,
+  closeIcon: <span aria-label="close icon">x</span>,
```

Comment:
I think we do not need the `closeIcon` to be one of the storybook args as it accepts a ReactNode and it is hard to configure this in the storybook UI